.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    strategy:
+      matrix:
+        go-version: [ 1.17.x ]
+        task: [ "lint", "test" ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: zendesk/checkout@v2
+      - name: Setup Golang
+        uses: zendesk/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Setup cache
+        uses: zendesk/cache@v2
+        with:
+          path: /home/runner/.cache/go-build
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: golangci-lint
+        uses: zendesk/golangci-lint-action@v2
+        with:
+          version: v1.42
+      - name: go ${{ matrix.go-version }} target ${{ matrix.task }}
+        run: OPERATOR=scaling make ${{ matrix.task }}
```

Comment:
can OPERATOR be set by default ? ... are there other valid values for it ?

Reply:
we could set it up that way but that's how the upstream had theirs setup